### ISSUE TITLE
Arch tool

### ISSUE THREAD

**Issue by joewintergreen** [gabrielw-us/ProBuilder-Unreal-Public-Tracker#82]
It's in Probuilder on Unity, so I would assume it's in the worse, but you gotta have that arch tool.

**Comment by arthursw** [comment]
yes, I planned to have one or two "angle" parameter for the tube tool, which would be the equivalent of the arch tool (but we can also have an arch tool, with more parameters and different orientation).

**Comment by gabrielw-us** [comment]
Ah, good point! I had wondered about that before. Arch is just a tube with 4 sides...

However, I think most people will not see this. I'd vote for a separate "Arch" tool, even if it is technically a duplicate. Also just for simplicity's sake.
